build.gradle

```diff
@@ -210,13 +227,19 @@ dependencies {
     exclude group: "org.hamcrest", module: "hamcrest-core"
     exclude group: "org.hamcrest", module: "hamcrest-library"
   }
+  testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.0.7'
+  testCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
```

Comment:
Did is not work without `serenity` ?

Reply:
Removed serenity and used only RestAssured